packages/dnb-eufemia/src/extensions/forms/hooks/useFieldProps.ts

```diff
@@ -153,28 +153,18 @@ export default function useFieldProps<
     showFieldError: showFieldErrorFieldBlock,
     mountedFieldsRef: mountedFieldsRefFieldBlock,
   } = fieldBlockContext ?? {}
-  const inIterate = Boolean(iterateElementContext)
   const {
     index: iterateElementIndex,
     path: iteratePath,
     handleChange: handleChangeIterateContext,
   } = iterateElementContext ?? {}
   const { setFieldError } = fieldBoundaryContext ?? {}
 
-  if (path && path.substring(0, 1) !== '/') {
-    throw new Error(
-      'Invalid path. Data value path JSON Pointers must be from root (starting with a /).'
-    )
+  if (path && path[0] !== '/') {
```

Comment:
```suggestion
  if (path && path.startsWith('/')) {
```

Not a necessary change, but maybe we could replace this with `startsWith `to make it more readable?

not that `path[0] !== '/'` is unreadable when we now that path is a `string `though!